target/linux/bmips/files/drivers/net/ethernet/broadcom/bcm6368-enetsw.c

```diff
@@ -251,32 +254,35 @@ static inline void dmas_writel(struct bcm6368_enetsw *priv, u32 val,
 /*
  * refill rx queue
  */
-static int bcm6368_enetsw_refill_rx(struct net_device *dev)
+static int bcm6368_enetsw_refill_rx(struct net_device *dev, bool napi_mode)
 {
 	struct bcm6368_enetsw *priv = netdev_priv(dev);
 
 	while (priv->rx_desc_count < priv->rx_ring_size) {
 		struct bcm6368_enetsw_desc *desc;
-		struct sk_buff *skb;
-		dma_addr_t p;
 		int desc_idx;
 		u32 len_stat;
 
 		desc_idx = priv->rx_dirty_desc;
 		desc = &priv->rx_desc_cpu[desc_idx];
 
-		if (!priv->rx_skb[desc_idx]) {
-			skb = netdev_alloc_skb(dev, priv->rx_skb_size);
-			if (!skb)
+		if (!priv->rx_buf[desc_idx]) {
+			unsigned char *buf;
+
+			if(likely(napi_mode))
```

Comment:
nitpick space after if.

Reply:
Done

Reply:
Mhhh think you misunderstood :D
Was referring to
```
if(likely(napi_mode))
```
that should be changed to
```
if (likely(napi_mode))
```

not adding an extra space after
```
if (!priv->rx_buf[desc_idx]) {
```
(that was fine)

Reply:
Done